hw/ip/otbn/doc/_index.md

```diff
@@ -811,14 +811,16 @@ Passing data between the host CPU and OTBN is done through the data memory (DMEM
 No standard or required calling convention exists, every application is free to pass data in and out of OTBN in whatever format it finds convenient.
 All data passing must be done when OTBN is not running, as indicated by the {{< regref "STATUS.busy" >}} bit; during the OTBN operation both the instruction and the data memory are inaccessible from the host CPU.
 
-## Returning from an application
+## Returning from an application {#writing-otbn-applications-ecall}
 
 The software running on OTBN signals completion by executing the {{< otbnInsnRef "ECALL" >}} instruction.
 
-When it executes this instruction, OTBN:
-- Stops fetching and executing instructions.
-- Sets {{< regref "INTR_STATE.done" >}} and clears {{< regref "STATUS.busy" >}}, marking the operation as completed.
-- Writes zero to {{< regref "ERR_BITS" >}}.
+When it executes this instruction, OTBN performs the following actions:
```

Comment:
I think this sentence needs rephrasing slightly. Before the patch, this introduced a list of things that OTBN did. Now, it introduces a list of things that happen, so maybe "Once OTBN has executed this instruction, the following things happen"?